cmd/util_test.go

```diff
@@ -0,0 +1,13 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TesttextToGitlabText(t *testing.T) {
```

Comment:
```suggestion
func Test_textToGitlabText(t *testing.T) {
```
This won't run without an uppercase following `Test`, however `_` counts for this.

Reply:
Fixed.